lib/docs/scrapers/gnu/gcc.rb

```diff
@@ -46,79 +46,89 @@ class Gcc < Gnu
       'Wtrigraphs.html' => 'Invocation.html'
     }
 
+    version '10' do
+      self.release = '10.2.0'
+      self.base_url = "https://gcc.gnu.org/onlinedocs/gcc-#{release}/gcc/"
+    end
+
+    version '10 CPP' do
+      self.release = '10.2.0'
+      self.base_url = "https://gcc.gnu.org/onlinedocs/gcc-#{release}/cpp/"
+    end
+
     version '9' do
-      self.release = '9.2.0'
+      self.release = '9.3.0'
       self.base_url = "https://gcc.gnu.org/onlinedocs/gcc-#{release}/gcc/"
     end
 
     version '9 CPP' do
-      self.release = '9.2.0'
+      self.release = '9.3.0'
       self.base_url = "https://gcc.gnu.org/onlinedocs/gcc-#{release}/cpp/"
 
       options[:replace_paths] = CPP_PATHS
     end
 
     version '8' do
-      self.release = '8.3.0'
+      self.release = '8.4.0'
```

Comment:
Updating all previous releases requires (me) to download and re-generate all docs (for a consistent state). Unless there are substantial changes to older versions, please refrain from updating those versions (maybe except for the second to newest one). Thank you!